Islandora-CLAW/Alpaca
# Parallel tests

**Parallelize Alpaca tests**: (https://github.com/Islandora-CLAW/CLAW/issues/712)

# What does this Pull Request do?

Parallelizes the tests in `alpaca/islandora-indexing-fcrepo` by route.

# What's new?

Splits tests up into classes by route being tested and runs them in parallel.

An additional commit is included showing a possible shortening of test setup patterns. If it looks attractive, I can use it more widely.

# How should this be tested?

This PR should not change the functional behavior of Alpaca _at all_. It should speed up build time by some small amount.

# Interested parties
@Islandora-CLAW/committers
